### DISCUSSION TITLE
Create 'Style_guide.rst' and 'testing.rst' files

### DISCUSSION THREAD

**Discussion by miertla** [beetbox/beets#4783] [General]
As I was reading over the documentation I noticed that the style guide and testing documentation is in the contributing document and I was wondering if it would make more sense to create a 'Style_guide' file and a'Testing' file for easier reference after a potential contributor read over this document.  I'm brand new to open source software and am currently in the process of getting my computer science degree so I'm learning the norms of open source in my current class so if this is something that is commonly done then I will take this as a learning experience. Thank you!

**Comment by JOJ0** [comment]
Do you have some links to what a typical Testting and Style_guide file would include.

If I'm assuming correctly a Testing file would explain how to new contributors how to run and write unittests. That correct? At least the "how to run single unittests" is something I would like to include in the docs.

If you want to contribute keep in mind that big changes might not get merged any time soon. Iterative changes are easier for us to review so maybe you might find a better approach than "reinventing the wheel" / splitting up a lot of things.

Improvment to the docs is always welcome though! Please elaborate what you have in mind. Thanks.
